widgets/common/src/DynamicDropdown.jsx

```diff
@@ -202,7 +214,8 @@ DynamicDropdown.defaultProps = {
     prefetch: false,
     multiple: false,
     className: '',
-    refreshEvent: null
+    refreshEvent: null,
+    itemsFormatter: items => items
```

Comment:
I'd reuse already existing function:
```suggestion
    itemsFormatter: _.identity
```